Makefile

```diff
@@ -146,7 +146,7 @@ translation-django-makemessages: assets
 	python -m kolibri manage makemessages -- -l en --ignore 'node_modules/*' --ignore 'kolibri/dist/*'
 
 translation-django-compilemessages:
-	python -m kolibri manage compilemessages
+	cd kolibri && PYTHONPATH="..:$$PYTHONPATH" python -m kolibri manage compilemessages
```

Comment:
mind adding a comment explaining why this is necessary in another PR?

Reply:
I wasn't actually able to state this in a convincing manner -- it seems that kolibri isn't installed with the `pip -e .` or similar in the Docker builds, so they are dependent on the path environment to import the `kolibri` module. This part, I find should be changed, but I'm no Docker expert.

In order to compile *ONLY* the messages in `kolibri/`, we have to change working dir and add the parent to the path.

Reply:
Comment added here: https://github.com/learningequality/kolibri/pull/3260